deploy/charts/openebs-monitoring/values.yaml

```diff
@@ -120,6 +120,12 @@ kube-prometheus-stack:
       - grafana-polystat-panel
       - snuids-trafficlights-panel
 
+customDashboards:
+  cStor:
```

Comment:
`cstorPools` instead of `cStor` because volumes dashboards are common for both cStor and jiva but pools are specific to cstor pools.

Reply:
For dashboards, naming is done according to the storage engine.  For alertRules, naming is done with respect to pools and volumes. That's why its `cStor`.

Reply:
Okay.